arm/Microsoft.ServiceBus/namespaces/disasterRecoveryConfigs/deploy.bicep

```
@@ -9,10 +9,10 @@ param name string = 'default'
 @description('Optional. Primary/Secondary eventhub namespace name, which is part of GEO DR pairing')
 param alternateName string = ''
 
-@description('Optional. ARM Id of the Primary/Secondary eventhub namespace name, which is part of GEO DR pairing')
+@description('Optional. ARM ID of the Primary/Secondary eventhub namespace name, which is part of GEO DR pairing')
 param partnerNamespace string = ''
```

ARM ID? Is this Resource ID? 
```suggestion
@description('Optional. Resource ID of the Primary/Secondary event hub namespace name, which is part of GEO DR pairing')
param partnerNamespaceResourceID string = ''
```